common/ECChannel.cpp

```diff
@@ -89,7 +90,7 @@ HRESULT ECChannel::HrSetCtx(ECConfig *lpConfig) {
 		OPENSSL_init_ssl(OPENSSL_INIT_ENGINE_ALL_BUILTIN, NULL);
 
 	// Default TLS context for OpenSSL >= 1.1, enables all methods.
-	auto newctx = SSL_CTX_new(TLS_server_method());
+	ctx = SSL_CTX_new(TLS_server_method());
```

Comment:
It's for extracting the variable initialization to an earlier point, so that goto with a single exit works; otherwise you can't use goto exit above the context creation. But this is pretty irrelevant in either way.